syntax = "proto3";

package bakdata.common.error.v1;

message CrawlError {
  string id = 1;
  string cause = 2;
}
